Changes

```diff
@@ -1,5 +1,54 @@
 Release history of Zonemaster
 
+v2025.2 2025-12-17
+
+ [Release information]
+- The GUI in this release has been completely rewritten. Customization has
+  been simplified. See Zonemaster-GUI.
+- Translations have not been fully updated in this release. They will be
+  updated in an upcoming extra release.
+- A zone (domain name) without DNSSEC will from this release create a
+  WARNING message, not just a message on NOTICE level (#1425)
+
+ [Breaking changes]
+ - Removes deprecated API method get_batch_job_result. See #1433 and
+   Zonemaster-Backend.
+ - Customization in GUI of previous version requires migration for this
+   version. See Zonemaster-GUI.
+ - The Zonemaster URL API has changed. See "using GUI" documentation.
```

Comment:
I'm wondering if we could just provide direct links to the actual documents in such sentences. What do you think?

Reply:
The links to doc.zonemaster.net are much shorter. I felt the other ones will be too long. In "Changes" the must be visiable. In "RELEASE.md" they could be hidden under some other link name but then the two files are more different than they should be.

Compare:
```
https://doc.zonemaster.net/v2025.2/configuration/gui/README.md
https://github.com/zonemaster/zonemaster/blob/master/docs/public/configuration/README.md
```